cram/cram_codecs.c

```diff
@@ -558,7 +558,7 @@ cram_codec *cram_beta_decode_init(char *data, int size,
     else if (option == E_BYTE_ARRAY || option == E_BYTE)
 	c->decode = cram_beta_decode_char;
     else {
-	fprintf(stderr, "BYTE_ARRAYs not supported by this codec\n");
+	hts_log_error("BYTE_ARRAYs not supported by this codec");
```

Comment:
I'd be happy with rationalising the spacing throughout, besides there are usually white space options for diffing files.

Frankly the only reason I had a mix of tabs and spaces is simply because it's the emacs default and it's invisible (plus tab = 8 space is standard on Unix) so I never really cared!  Things have moved on a bit though and it's probably time to coax the code into pure space mode.

I agree it's best done in its own pull request though, and keeping the same formatting as current code is good for now.